R/get_all_model_abbr.R

```diff
@@ -0,0 +1,19 @@
+#' Get all valid model_abbrs from remote hub repo
```

Comment:
could we add an option to do this from zoltar as an alternative?

Reply:
Are we still calling these `model_abbr`'s, or just `model`s?  (I'm not sure)